### DISCUSSION TITLE
2022-06-01 Weekly Meeting 58 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#239] [Weekly Meetings]
Weekly Meeting 58 is happening on Wednesday June 1st, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

## Announcements: 
Any demos or things to share from the community? 

Working Group Updates:
- [OMI Website update](https://github.com/omigroup/website)
  - Blog update
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - Second OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- any others?

**Comment by mrmetaverse** [comment]
Elijah with Zesty shared that last week they noticed a 15-20% increase in webxr traffic (mostly due to pico neo 3). More headsets coming online are contributing to an increase overall of traffic.

**Comment by mrmetaverse** [comment]
# Website 

- posts in review
- Blog going on line hopefully soon

**Comment by mrmetaverse** [comment]
# Scripting group - Developer Hour! 
- started off with beginner topics, helping people with normal JS best practices (don't get to do that often!) 
- Dove into computer graphics topics, godot engine topics, threejs topics, and more. 
- Really fun conversation 
- next Dev open hour is in two weeks (bi-weekly)

**Reply by mrmetaverse** [reply]
If people are interested we can run the next one as a Twitter space! We don't do many events on twitter, but the OMI dev hour feels appropriate for this format next.

**Comment by mrmetaverse** [comment]
Consider more open hours for your working groups as well. Design open hour, Avatar open hour, Blender open hour, etc.

**Comment by mrmetaverse** [comment]
# gLTF extensions 
- https://github.com/omigroup/gltf-extensions
- discussing authoring tools for a while 
- found some gaps in gLTF related to using it as an interoperable authoring format 
- proposed OMI-ref which allows embedding other gLTF documents in a gLTF 
- test with a stylized nature pack compiling all assets into a single gLTF, and link to the different pieces of it using the OMI-ref tool. 

Robert said this would work really well for the existing spoke editor, also building a new editor in element. 

Also some new developments with godot engine 
- import/export workflow for hubs content into godot (import spoke assets into godot and use them there) 
- also export quite a few things from godot, getting closer to an interoperable pipeline for importing and exporting 

## Looking for help 
seeking others who want to be involved in the dev and implementation. 

Reminds us that the prior week we were demoing a Lyuma piece for importing and exporting using godot. 

Ant was using spokes media frames as a way to author gLTF content that could be controlled with threejs, replacing those. If you want to replace artwork in a 3d scene, you could easily use that. Spoke needs some way to add gLTF exif data (arbitrary JSON data that can be attached to different nodes in a scene graph) then we could author any sort of content we want using spoke or related tooling. 

RE: KHR audio 
The hubs team got back to us. they are looking to add support, but are in the middle of a refactor. We are blocked on getting in there, but the sentiment is positive so it could be sooner than later.

**Reply by mrmetaverse** [reply]
Q: Would the OMI-ref be able to refer to the assets from a URL as its source too? 
A: _ANY_ URI that gLTF supports. By default they only support relative pass, but this spec opens up other possibilities based on your ecosystems needs.

**Reply by robertlong** [reply]
Q: Does glTF support bounding boxes?
A: glTF accessors have min/max properties that can be used to quickly generate bounding boxes. We're also working on a colliders spec so you can use those in your app as well.

**Comment by mrmetaverse** [comment]
# Media and Content Strategy Working group
- suggested splitting working group into different streams 
  - content 
  - process automation 
  - PR 
- In the next meeting, Shirley hopes to present draft of how we can connect the dots and maximize the content we put out to some extent. 
- with respect to the progression: One idea was to try and push whatever was published from twitter into one channel so anyone can easily like, share, and get involved with threads on twitter. Shirley found a bot for that and will need help getting it implemented.

**Reply by mrmetaverse** [reply]
Twitter post bot

**Comment by mrmetaverse** [comment]
# Lets get first blog post published this week!

**Reply by mrmetaverse** [reply]
cc: @antpb @pcsmgnt @LightLodges

**Reply by mrmetaverse** [reply]
need final read from one reviwer then we are publishing today!! :rocket:

**Comment by mrmetaverse** [comment]
REVIEW NEXT WEEK! https://github.com/omigroup/omigroup/issues/238

**Comment by mrmetaverse** [comment]
# Lean Coffee session from 06/01/22
[OMI-lean-coffee-june-1.pdf](https://github.com/omigroup/omigroup/files/8863632/OMI-lean-coffee-june-1.pdf)
